src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -106,7 +113,7 @@ public void evaluate() throws Throwable {
                     base.evaluate();
                 }
 
-                private List<PluginSpec> combinePlugins(WithPlugins... wp) {
+                private List<PluginSpec> combinePlugins(List<WithPlugins> wp) {
```

Comment:
should this not be be a set to prevent the case where people use the annotation on both the parent and the current and have set it both the same to work around this issue.